### PR TITLE
[terra-modal-manager] Reenabling wdio tests

### PR DESCRIPTION
**Pull request by BenBoersma**
### Summary
Enabling wdio tests and adding in missing screenshots for orion-fusion-theme

Thanks for contributing to Terra.
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-modal-manager/tests/wdio/modal-manager-spec.js [hunk 1]
```diff
@@ -492,7 +492,7 @@ Terra.describeViewports('ModalManager - Disclosure Accessory', ['large'], () =>
 
     it('renders each disclosed component in a disclosure container', () => {
       expect(browser.isExisting('[data-disclosure-container="true"]')).to.be.true;
-      Terra.validates.element('nested disclosure', { selector });
+      Terra.validates.screenshot('nested disclosure', { selector });
```
Comment (emilyrohrbough) [inline packages/terra-modal-manager/tests/wdio/modal-manager-spec.js]: Can you remove the `only` on this describe block? It was something I accidently merged in

Comment (BenBoersma) [reply packages/terra-modal-manager/tests/wdio/modal-manager-spec.js]: removed here https://github.com/cerner/terra-framework/pull/989/commits/dc3742c06e6b79e7055729172d004837736b891a

**Comment by BenBoersma** [comment]
I ran master with the new axe-core update changes and am no longer getting the wdio failure this PR fixes. Since the error was intermittent, I ran the tests locally about 4+ times, and it succeeded on every one. 

However this PR still has the fix for removing the `describe.only` so I can undo the tests changes from `.element` to `.screenshot` and keep the the change to `describe.only` and the screenshots that produces.  @neilpfeiffer @ryanthemanuel thoughts?

**Comment by BenBoersma** [comment]
As per offline discussion, Changing this PR to just reenabling some wdio tests and adding the missing orion-fusion-theme screenshots for those tests.

**Pull request merged**